tutorials/npm/src/js/runtime.mjs

```diff
@@ -0,0 +1,18 @@
+import { RenderInfo, lib } from "icu4x/demo";
+import { TerminusRender } from "icu4x/demo/rendering";
+
+let params = new URLSearchParams(window.location.search);
+
+let func = params.get("func");
+
+const dataProvider = lib.DataProvider.compiled();
+
+let terminus = new TerminusRender(lib, (param, updateParamEvent) => {
```

Comment:
@sffc our demo was under tutorials/, and was an example of using Diplomat from JS

an autogenerated demo is of course not a good tutorial for use of Diplomat. We want to publish the demo as well, should we have it live elsewhere, or something else? (and should we keep around the old demo as a "tutorial", or is it low-value?)

Reply:
Ah, I didn't realize we wanted to keep both in. I was using a different folder under tutorials/ anyways, so I could restore that. I'm also curious as to where we want to put this, though.

Reply:
To be clear, I'm not sure if we want to keep both!

What I notice is that right now we have a tutorial that is actually helpful for learning to use the JS backend, and is a demo of okayish but not amazing quality. The Demo is _way_ better as a demo since it covers basically everything, however it is not a tutorial.

Important questions to get answered:

 - Do we _need_ a JS tutorial?
 - Is it acceptable for The Demo to live under tutorials? If not, where?
 - Do we need both the demo and the js tutorial?


All that said, we definitely _want_ to land this demo, so I don't consider this a huge blocker and tend towards merging this. We can resurrect the old demo from git history.

cc @robertbastian as well

Reply:
I'm fine keeping the previous demo code solely as a _tutorial example_, so long as the main demo published on the web site uses the new code.